CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ The format can be described more formally as follows:
 
 ## Architecture
 
-The Dashboard application consists of 2 parts:
+The web client application consists of 2 parts:
```

Comment:
Should we use `bblfsh web clinet` for the context instead of just `web client`?

Reply:
Imo it's unnecessary. If you opened CONTRIBUTING file of the project you most probably know what it does. In this case, you know what it's client for. We also mention babelfish in this document 6 times already. 
Though I don't mind to change to `bblfsh web client`, please confirm once again we really need this change.

Reply:
That is reasonable, so to change or not it's up to you. And thanks for double-checking!

Reply:
I changed it on the first appearance in this document to make it more clear. But below I left it just as `web client`.